src/content/getting-started/prepare-your-provider-account/azure/_index.md

```diff
@@ -3,11 +3,135 @@ title: Prepare your provider account for Azure
 linkTitle: Prepare your Azure account
 description: Prepare your Azure account to start building your cloud-native developer platform with Giant Swarm.
 weight: 20
-last_review_date: 2024-04-05
+last_review_date: 2024-05-28
 owner:
   - https://github.com/orgs/giantswarm/teams/sig-docs
 user_questions:
   - How do I prepare my Azure account for the cloud-native developer platform?
   - What do I need to do to prepare my Azure account for the cloud-native developer platform?
 ---
 
+In order to run the Giant Swarm platform in your Azure subscriptions, several prerequisites must be satisfied to support Cluster API Provider for Azure (CAPZ).
+
+## Requirements
+
+To create and assign the role to Giant Swarm's Service Principal you need:
+
+- An account with [Owner](https://docs.microsoft.com/en-us/azure/role-based-access-control/built-in-roles#owner) or [User Access Administrator](https://docs.microsoft.com/en-us/azure/role-based-access-control/built-in-roles#user-access-administrator) role.
+- [Azure CLI](https://docs.microsoft.com/en-us/cli/azure/install-azure-cli) installed.
+
+### Service quotas {#quotas}
+
+<!-- DO WE NEED TO DO THAT BY DEFAULT LIKE IN AWS? -->
```

Comment:
Reading from the checklist, we reference a page with limits in Azure, but do our customers bump the default limits for any of the resources?